controller/productController.js

```diff
@@ -0,0 +1,102 @@
+const productService = require('../services/productService')
+const middleJwt = require('../middleware/jwt');
+
+const productAll = async(req,res) =>{
+    try{
+        const {brandName,scentName} = req.query
+        const products = await productService.getProducts(brandName,scentName)
+        res.status(200).json({products})
+    }catch(err){
+        res.status(500 || statusCode).json({message: err.message})
+    }
+}
+
```

Comment:
```suggestion

const productAll = async(req,res) =>{
    try {
        const { brandName, scentName } = req.query
        const products = await productService.getProducts(brandName, scentName)
        
        return res.status(200).json({products})
    } catch(err) {
        res.status(statusCode || 500).json({ message: err.message })
    }
}

```
`500 || statusCode` 로 하면, 모든 에러가 100% 500으로 코드가 갑니다. 왜일까요?

Reply:
try -catch 띄어쓰기 간격이 없어요. 구문 앞뒤로 공백 한 칸씩 띄워주시고, 모든 파일에 전면 반영해 주세요 :)

Reply:
OR 조건 때문에 500이 들어오면 항상 true로 판단하기 때문에 500으로 코드가 나가는 것 같습니다.
소헌님이 말씀해주신 코드대로 변경을 하면 statusCode가 존재하면 해당 값이 사용되고 그렇지 않으면 500이 사용 되는것 같습니다..(?)